algorithm/cgal/Bend.h

```diff
@@ -3,49 +3,128 @@
 #include "demesh_util.h"
 #include "point_util.h"
 
-static int Bend(Geometry* geometry, double reference_radius) {
+static int Bend(Geometry* geometry, double reference_radius,
```

Comment:
What does bend do again?

Reply:
Applies a uniform curvature to the shape.

Effectively placing it along the perimeter of a circle of the given radius.

It's not quite what we generally want, though.